docs/development.md

```diff
@@ -0,0 +1,32 @@
+# Build and Development
+
+This project supports Python and can be built from source easily, or a simple cmake build without Python dependency.
+## Python package
+The package contains all custom operators and some Python scripts to manipulate the ONNX models.
+- Install Visual Studio with C++ development tools on Windows, or gcc(>8.0) for Linux or xcode for macOS, and cmake on the unix-like platform. (**hints**: in Windows platform, if cmake bundled in Visual Studio was used, please specify the set _VSDEVCMD=%ProgramFiles(x86)%\Microsoft Visual Studio\<VERSION_YEAR>\<Edition>\Common7\Tools\VsDevCmd.bat_)
+- If running on Windows, ensure that long file names are enabled, both for the [operating system](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd) and for git: `git config --system core.longpaths true`
```

Comment:
When enabling long paths for the operating system, this linked site was useful but I ran into some issues and the directions on this other site helped: https://learn.microsoft.com/en-us/answers/questions/730467/long-paths-not-working-in-windows-2019.html

We could add this as well for user reference.

Reply:
Feel free to create a PR to enhance the section in this document. I can only share what I knew here.